pkg/cloudprovider/providers/azure/azure_blobDiskController.go

```diff
@@ -519,7 +519,7 @@ func (c *BlobDiskController) createStorageAccount(storageAccountName string, sto
 			return fmt.Errorf("azureDisk - can not create new storage account, current storage accounts count:%v Max is:%v", len(c.accounts), maxStorageAccounts)
 		}
 
-		glog.V(2).Infof("azureDisk - Creating storage account %s type %s \n", storageAccountName, string(storageAccountType))
+		glog.V(2).Infof("azureDisk - Creating storage account %s type %s\n", storageAccountName, string(storageAccountType))
```

Comment:
what has this PR to do with `azure_blobDiskController.go`? only delete a space? It's ok but not related to this PR...

Reply:
just follow [https://github.com/kubernetes/kubernetes/pull/56409](https://github.com/kubernetes/kubernetes/pull/56409), i see also in  `/provider` directory, just fix it.

Reply:
Thanks,  i will remove it